superset/connectors/sqla/models.py

```diff
@@ -1333,7 +1335,9 @@ def fetch_metadata(self, commit: bool = True) -> MetadataResult:
         db_engine_spec = self.database.db_engine_spec
         old_columns = db.session.query(TableColumn).filter(TableColumn.table == self)
 
-        old_columns_by_name = {col.column_name: col for col in old_columns}
+        old_columns_by_name: Dict[str, TableColumn] = {
+            col.column_name: col for col in old_columns
+        }
```

Comment:
Not sure if it's just my IDE, but PyCharm wasn't able to infer the type without the explicit type hint.